src/Symfony/Bridge/PhpUnit/ClockMock.php

```diff
@@ -73,7 +74,7 @@ public static function register($class)
     {
         $self = \get_called_class();
 
-        $mockedNs = array(substr($class, 0, strrpos($class, '\\')));
+        $mockedNs = [substr($class, 0, strrpos($class, '\\'))];
```

Comment:
Must be reverted.

Reply:
Thank you but the PR is not ready, it's just a test to trigger and observe CI behavior :)